src/error.js

```diff
@@ -327,6 +327,7 @@ export function getErrorReportUrl(message, filename, line, col, error,
   // for analyzing production issues.
   let url = urls.errorReporting +
       '?v=' + encodeURIComponent('$internalRuntimeVersion$') +
+      '&rtv=' + encodeURIComponent(getMode().rtvVersion) +
```

Comment:
This is fine for backwards compat, but any chance we could get rid of `v` eventually?